validation/src/main/scala/hmda/validation/engine/lar/ULI.scala

```diff
@@ -0,0 +1,65 @@
+package hmda.validation.engine.lar
+
+//See https://www.consumerfinance.gov/eregulations/1003-C/2015-26607_20180101#1003-C-1
+
+object ULI {
+
+  val conversionTable = Map(
+    "a" -> 10,
+    "b" -> 11,
+    "c" -> 12,
+    "d" -> 13,
+    "e" -> 14,
+    "f" -> 15,
+    "g" -> 16,
+    "h" -> 17,
+    "i" -> 18,
+    "j" -> 19,
+    "k" -> 20,
+    "l" -> 21,
+    "m" -> 22,
+    "n" -> 23,
+    "o" -> 24,
+    "p" -> 25,
+    "q" -> 26,
+    "r" -> 27,
+    "s" -> 28,
+    "t" -> 29,
+    "u" -> 30,
+    "v" -> 31,
+    "w" -> 32,
+    "x" -> 33,
+    "y" -> 34,
+    "z" -> 35
+  )
+
+  def convert(loanId: String): String = loanId
+    .map(_.toLower)
+    .map { c =>
+      if (!c.isDigit)
+        conversionTable(c.toString).toString
```

Comment:
Would it be better to do a conversion based on the ASCII value?  Something like `c.hashCode - 87`.  Though, a downside is that it's not immediately obvious that it follows the regulation.

Reply:
Yeah, that's a good suggestion, but I think it obscures the intent a bit. I'm ok with making this change if you think it's worth it.

Reply:
Definitely not a necessity, I'm ok with leaving it as-is for the transparency.